Index: lesson_5/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.idea/\r\n*.iml\r\nout/\r\n*.class
===================================================================
diff --git a/lesson_5/.gitignore b/lesson_5/.gitignore
--- a/lesson_5/.gitignore	(revision a5284b46a98c0f2cdee7f574a4e9a9e3bd43e7bd)
+++ b/lesson_5/.gitignore	(date 1791498124293)
@@ -1,4 +1,3 @@
 .idea/
 *.iml
 out/
-*.class
\ No newline at end of file
